samples/dfu/updates/.gitignore

```diff
@@ -0,0 +1 @@
+*.bin
```

Comment:
You don't need this if you build inside "build" directories, which one typically does.